Tools/setup/Dockerfile

```diff
@@ -0,0 +1,55 @@
+#
+# PX4 base development environment
+#
+
+FROM ubuntu:20.04
+LABEL maintainer="Daniel Agar <daniel@agar.ca>, Ramon Roche <mrpollo@gmail.com>"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+
+# Installing required utilities
+RUN apt-get update && apt-get -y --quiet --no-install-recommends install \
+  ca-certificates \
+  gnupg \
+  lsb-core \
+  lsb-release \
+  sudo \
+  software-properties-common \
+  wget \
+  gosu \
+  ;
+
+# Install PX4 Requirements
+COPY Tools/setup/requirements.txt /tmp/requirements.txt
+COPY Tools/setup/ubuntu.sh /tmp/ubuntu.sh
```

Comment:
```suggestion
COPY Tools/setup/requirements.txt Tools/setup/ubuntu.sh gcc-arm-none-eabi-9-2020-q2-update-linux.tar.* /tmp/
```

Reply:
This allows a 'graceful failure' if the gcc tarball isn't found

Reply:
Oddly, the GCC tarball copy is failing on your end with the pattern matching with the star prepend, but I can def change that if it works for you.

Reply:
If it fails for me, I doubt I'm the only one; this tweak did the trick for me though 👍🏻

Reply:
I made a change on the PR, I still like the 3 COPY statements so we can CACHE those files independently, there are times when you are testing a build, and you didn't change one of the files its nice to pull from CACHE

Reply:
It still fails for me... I think we need to add a dummy file so that Docker has at least one file to copy:

```
jacob@pop-os:~/Codes/Autopilot$ ./Tools/docker_build.sh
[docker_build.sh]: Building [px4io/px4-dev:latest]
[docker_build.sh]:  - with args: []
Sending build context to Docker daemon  2.576GB
Step 1/23 : FROM ubuntu:20.04
 ---> 20fffa419e3a
Step 2/23 : LABEL maintainer="Daniel Agar <daniel@agar.ca>, Ramon Roche <mrpollo@gmail.com>"
 ---> Using cache
 ---> d07232564940
Step 3/23 : ENV DEBIAN_FRONTEND noninteractive
 ---> Using cache
 ---> a3f07effde5e
Step 4/23 : ENV LANG C.UTF-8
 ---> Using cache
 ---> 099bf4d64952
Step 5/23 : ENV LC_ALL C.UTF-8
 ---> Using cache
 ---> 73e3e5afe39f
Step 6/23 : RUN apt-get update && apt-get -y --quiet --no-install-recommends install   ca-certificates   gnupg   lsb-core   lsb-release   sudo   software-properties-common   wget   gosu   ;
 ---> Using cache
 ---> 91ac6831e57f
Step 7/23 : COPY Tools/setup/requirements.txt /tmp/requirements.txt
 ---> Using cache
 ---> 64997578eada
Step 8/23 : COPY Tools/setup/ubuntu.sh /tmp/ubuntu.sh
 ---> d06a4bdf7fa2
Step 9/23 : COPY gcc-arm-none-eabi-9-2020-q2-update-linux.tar.* /tmp/
COPY failed: no source files were specified
```

Reply:
I added an empty dummy.file to the setup folder for this purpose; does the trick for me.

Reply:
Can we drop this entirely? Obviously not having to download again speeds up the "Dockerfile build" iteration speed, but for any real usage isn't it fine?